README.md

```diff
@@ -20,62 +91,429 @@ Add the following to your `pubspec.yaml` file:
+  }
+);
+```
+
+### customRender:
+
+A powerful API that allows you to customize everything when rendering a specific HTML tag. This means you can add support for HTML elements that aren't supported natively. You can also make up your own custom tags in your HTML!
+
+`customRender` accepts a `Map<String, CustomRender>`. The `CustomRender` type is a function that requires a `Widget` to be returned. It exposes `RenderContext`, the `Widget` that would have been rendered by `Html` without a `customRender` defined, the `attributes` of the HTML element as a `Map<String, String>`, and the HTML element itself as `Element`.
+
+To use this API, set the key as the tag of the HTML element you wish to provide a custom implementation for, and create a function with the above parameters that returns a `Widget`.
+
+#### Example Usages - customRender:
+1. Simple example - rendering custom HTML tags
+<details><summary>View code</summary>
+
+```dart
+Widget html = Html(
+  data: """
+  <h3>Display bird element and flutter element <bird></bird></h3>
+  <flutter></flutter>
+  <flutter horizontal></flutter>
+  """,
+  customRender: {
+      "bird": (RenderContext context, Widget child, Map<String, String> attributes, _) {
+        return TextSpan(text: "🐦");
       },
-      onImageTap: (src) {
-        // Display the image in large form.
+      "flutter": (RenderContext context, Widget child, Map<String, String> attributes, _) {
+        return FlutterLogo(
+          style: (attributes['horizontal'] != null)
+              ? FlutterLogoStyle.horizontal
+              : FlutterLogoStyle.markOnly,
+          textColor: context.style.color,
+          size: context.style.fontSize.size * 5,
+        );
       },
+    },
+);
+```
+</details>
+
+2. Complex example - rendering an `iframe` differently based on whether it is an embedded youtube video or some other embedded content
+
+Packages used: [`data_connection_checker`](https://pub.dev/packages/data_connection_checker) and [`flutter_inappwebview`](https://pub.dev/packages/flutter_inappwebview)
+
+<details><summary>View code</summary>
+
+```dart
+class CustomRenderExample extends StatefulWidget {
+  CustomRenderExample({Key key, this.title}) : super(key: key);
+  
+    final String title;
+  
+    @override
+    CustomRenderExampleState createState() => new CustomRenderExampleState();
+}
+
+class CustomRenderExampleState extends State<CustomRenderExample> {
+  bool result = false;
+  
+  @override
+  void initState() async {
+    // store current internet connection status
+    result = await DataConnectionChecker().hasConnection;
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: AppBar(
+        title: Text('flutter_html Example'),
+        centerTitle: true,
+      ),
+      body: SingleChildScrollView(
+        child: Html(
+             data: """
+             <h3>YouTube iframe:</h3>
```

Comment:
Minor, but you reversed the two examples here (the Google and YouTube iframes).

Reply:
Oops, fixed this!